static/app/views/performance/browser/webVitals/utils/queries/rawWebVitalsQueries/useTransactionRawWebVitalsQuery.tsx

```diff
@@ -40,11 +42,18 @@ export const useTransactionRawWebVitalsQuery = ({
         'p75(measurements.cls)',
         'p75(measurements.ttfb)',
         'p75(measurements.fid)',
+        'count_web_vitals(measurements.lcp, any)',
+        'count_web_vitals(measurements.fcp, any)',
+        'count_web_vitals(measurements.cls, any)',
+        'count_web_vitals(measurements.ttfb, any)',
+        'count_web_vitals(measurements.fid, any)',
         'count()',
       ],
       name: 'Web Vitals',
       query:
-        'transaction.op:pageload' + (transaction ? ` transaction:"${transaction}"` : ''),
+        'transaction.op:pageload' +
+        (transaction ? ` transaction:"${transaction}"` : '') +
+        (query ? ` ${query}` : ''),
```

Comment:
personally I think it looks cleaner with this as an array
```suggestion
       [
       'transaction.op:pageload',
       ...(transaction ? [`transaction:"${transaction}"`] : []),
       ...(query ? [`${query}`] : [])
       ].join(' ')
```